ui/src/components/connections/ConnectionDetailsRowDropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import { Dropdown, Row } from "antd";
+
+import { ReactComponent as IconDots } from "src/assets/icons/dots-vertical.svg";
+import { ReactComponent as IconInfoCircle } from "src/assets/icons/info-circle.svg";
+import { ReactComponent as IconTrash } from "src/assets/icons/trash-01.svg";
+import { ReactComponent as IconClose } from "src/assets/icons/x.svg";
+
+const MENU_ITEMS = [
+  {
+    icon: <IconInfoCircle />,
+    key: "details",
+    label: "Details",
+  },
+  {
+    key: "divider",
+    type: "divider",
+  },
+  {
+    danger: true,
+    icon: <IconClose />,
+    key: "revoke",
+    label: "Revoke",
+  },
+  {
+    key: "divider",
```

Comment:
We can't repeat this key.

![same-key-bug](https://user-images.githubusercontent.com/15896177/229789328-ca850e02-1b79-4316-8ec8-d08385817d9b.gif)